code/package.json

```diff
@@ -30,6 +30,7 @@
     "changelog": "pr-log --sloppy --cherry-pick",
     "changelog:next": "pr-log --sloppy --since-prerelease",
     "check": "NODE_ENV=production node ../scripts/check-package.js",
+    "check:scripts": "cd ../scripts && ./prepare/check-scripts.ts",
```

Comment:
With the `"nx"` field these can be picked up by `nx run` and `nx run-many`. Unfortunately, if we try to do this all in one command, the check task would also run `check-package` which I think we don't want it to do.

I considered renaming `check` to `check:package`, but then I would need to know everywhere it is used so it can be renamed, which is tricky using just text search.